vllm/core/block_manager_v1.py

```diff
@@ -15,6 +15,17 @@
 from vllm.utils import Device
 
 logger = init_logger(__name__)
+'''
+Exception strings for non-implemented encoder/decoder scenarios
+'''
+
+str_not_impl_enc_dec_swa = \
```

Comment:
make these uppercase since they are constants?

Reply:
Thanks! Fixed in most recent push.